vlingo-petclinic/src/test/java/io/vlingo/developers/petclinic/AnimalTypeEntityTests.java

```diff
@@ -0,0 +1,87 @@
+package io.vlingo.developers.petclinic;
+
+import io.vlingo.actors.World;
+import io.vlingo.actors.testkit.AccessSafely;
+import io.vlingo.actors.testkit.TestWorld;
+import io.vlingo.developers.petclinic.infrastructure.persistence.*;
+import io.vlingo.developers.petclinic.model.animaltype.*;
+import io.vlingo.lattice.model.sourcing.SourcedTypeRegistry;
+import io.vlingo.symbio.BaseEntry;
+import io.vlingo.symbio.EntryAdapterProvider;
+import io.vlingo.symbio.store.journal.Journal;
+import io.vlingo.symbio.store.journal.inmemory.InMemoryJournalActor;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AnimalTypeEntityTests {
```

Comment:
It appears that non of your tests test each of the factory methods in the protocol interfaces. Please add simple tests for each.

Reply:
What do you mean simple test for protocol interface? Isn't this protocol method calling?

https://github.com/hurelhuyag/vlingo-examples/blob/master/vlingo-petclinic/src/test/java/io/vlingo/developers/petclinic/AnimalTypeEntityTests.java#L72

Reply:
I meant the interface static factory methods.

Reply:
fixed by b7b25b44